dev/ProjectReunion_DLL/ProjectReunion_DLL.vcxproj

```diff
@@ -301,6 +306,7 @@
       <EnableUAC>false</EnableUAC>
       <AdditionalDependencies>onecore.lib;onecoreuap.lib;%(AdditionalDependencies)</AdditionalDependencies>
       <ModuleDefinitionFile>ProjectReunion.def</ModuleDefinitionFile>
+      <AdditionalOptions>/debug /debugtype:cv,fixup /opt:ref /opt:icf /incremental:no %(AdditionalOptions)</AdditionalOptions>
```

Comment:
Please change this to use a model like:

```xml
<AdditionalOptions>$(AdditionalPipelineLinkerOptions) %(AdditionalOptions)</AdditionalOptions>
```

Where that "additional pipeline" value is set by the ADO build pipeline as necessary. This lets us configure the linker commands during official build runs but not impact the individual developer's output.